### PR TITLE
build: change the sample values for ui templates

### PR DESCRIPTION
**Pull request by lorenzo-cavazzi**
These are the changes to the values file required by SwissDataScienceCenter/renku-ui#963

### CODE REVIEW

File: charts/values.yaml.changelog.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ Please follow this convention when adding a new row
 * `<type: NEW|EDIT|DELETE> - *<resource name>*: <details>`
 
 ----
+## Changes on top of Renku 0.6.5
+* DELETE - *ui.templatesRepository*
+* NEW - *ui.templates*: the UI project creation has been overhauled and the settings are stored here.
```
Comment (ableuler) [inline charts/values.yaml.changelog.md:10]:
```suggestion
* NEW - *ui.templates*: the UI project creation has been overhauled and can be configured here.
```

File: charts/values.yaml.changelog.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ Please follow this convention when adding a new row
 * `<type: NEW|EDIT|DELETE> - *<resource name>*: <details>`
 
 ----
+
+## Changes on top of Renku 0.7.0
```
Comment (ableuler) [inline charts/values.yaml.changelog.md:9]:
```suggestion
## Upgrading to Renku 0.8.0 (INCLUDES BREAKING CHANGES!)
```
I guess this should go into a `0.8.0` release as it requires changes to the values file, right?

File: charts/renku/values.yaml [hunk 3]
```diff
@@ -432,6 +432,13 @@ gitlab:
   ## Set the JupyterHub URL
   # jupyterhubUrl: "http://jupyterhub.renku.build"
 
+  ## Configuration for Renku templates
+  templates:
+    # custom: true
```
Comment (ableuler) [inline charts/renku/values.yaml:437]: I guess these defaults are already set in the UI `values.yaml`?

**Review by ableuler — CHANGES_REQUESTED** [review]

File: charts/renku/values.yaml (hunk 3 above)
Comment (lorenzo-cavazzi) [reply charts/renku/values.yaml:437]: Yes they are. Shouldn't I add `templates` anyway here?

File: charts/values.yaml.changelog.md (hunk 2 above)
Comment (lorenzo-cavazzi) [reply charts/values.yaml.changelog.md:9]: I thought the version 0.7.0 is yet to be released, right? Should I move these changes in the already existing 0.7.0 section?

Comment (ableuler) [reply charts/values.yaml.changelog.md:9]: ah yeah, ofc, sorry. Yes, just add it to the 0.7.0 section 👍

File: charts/renku/values.yaml (hunk 3 above)
Comment (ableuler) [reply charts/renku/values.yaml:437]: It's fine the way it is. I'd personally be in favour of not repeating all configuration options from the chart dependencies, but it's consistent with what we've been doing so far.

Comment (rokroskar) [reply charts/renku/values.yaml]: I was going to ask the same thing - do you need `templates` to be uncommented?

Comment (rokroskar) [reply charts/renku/values.yaml]: oops nvm I missed that you changed it already 😄

**Review by ableuler — APPROVED** [review]

**Pull request merged**